### PR TITLE
Seo updates April 28

### PR DESCRIPTION
**Pull request by mstekl**
I'm submitting a few set of improvements:

- Ran a fresh crawl and rebuilt sitemap.xml
- Fixed accessibility issues when using <button> on header and homepage
- Changed the text for links btns underuse cases section on homapage to a more descriptive text
- Replaced old 4.7 fontawesome with latest 5.11 which avoids font-display audit failure

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -273,9 +278,9 @@ <h3>Applications Acceleration &amp; Data Caching</h3>
                             using Ignite as an in-memory data grid.
                         </p>
 
-                        <a href="/use-cases/in-memory-data-grid.html" class="btn btn-primary" title="In-Memory Data Grid" 
+                        <a href="/use-cases/in-memory-data-grid.html"  role="button" class="btn btn-primary" title="In-Memory Data Grid" 
                            onclick="ga('send', 'event', 'apache_ignite_use_cases', 'homepage_click', 'data_grid');">
-                            Learn More</a>
+                           Learn More About In-Memory Data Grid</a>
```
Comment (dmagda) [inline index.html:283]: Please revert this change, that's too much for the button title. I believe we can sacrifice this SEO optimization in favor of better readability. Data Grid keyword is already mentioned three times on the front page.

File: index.html [hunk 2]
```diff
@@ -304,9 +309,9 @@ <h3>Digital Integration Hub</h3>
                             through Apache Ignite’s functionally rich, unified and consistent APIs.
                         </p>
 
-                        <a href="/use-cases/digital-integration-hub.html" class="btn btn-primary" title="Digital Integration Hub"
+                        <a href="/use-cases/digital-integration-hub.html"  role="button" class="btn btn-primary" aria-label="Learn More About Digital Integration Hub" title="Digital Integration Hub"
                            onclick="ga('send', 'event', 'apache_ignite_use_cases', 'homepage_click', 'digital_hub');">
-                            Learn More</a>
+                           Learn More About Digital Integration Hub</a>
```
Comment (dmagda) [inline index.html:314]: Please revert this change, that's too much for the button title. I believe we can sacrifice this SEO optimization in favor of better readability. Digital Hub keyword is already mentioned a couple of times on this page.

File: index.html (hunk 1 above)
Comment (mstekl) [reply index.html:283]: @dmagda, the purpose here was not to introduce a keyword, but to improve usability. It is a good usability practice to use descriptive links and avoid things like "click here", "read more", etc. 
Here is some literature: 
- https://accessibility.oregonstate.edu/descriptivelinks
- https://developers.google.com/web/tools/lighthouse/audits/descriptive-link-text

Also on this particular case I even think having a longer text on those buttons give more balance to the elements in terms of design. The button I propose is not even as long as the titles on those boxes, and they introduce more blue area that fills the empty space generated by the small "learn more" buttons.

I don't know which reasons are you basing to consider these as "too much for a button title", but please reconsider the merge based on this explanation. Thanks!
